rsc/src/main/java/com/cloudera/livy/rsc/driver/Statement.java

```diff
@@ -17,17 +17,19 @@
 
 package com.cloudera.livy.rsc.driver;
 
+import java.util.concurrent.atomic.AtomicReference;
+
 import com.fasterxml.jackson.annotation.JsonRawValue;
 
 public class Statement {
   public final Integer id;
-  public final StatementState state;
+  public final AtomicReference<StatementState> state;
   @JsonRawValue
-  public final String output;
+  public String output;
```

Comment:
Because we need to update the output later [here](https://github.com/cloudera/livy/pull/230/files#diff-726504ec4098560b7f98fdec97b165e7R98).